composer.json

```diff
@@ -20,26 +20,33 @@
         }
     },
     "require": {
-        "php": "^7.4|^8.0",
-        "react/socket": "^0.8",
-        "react/datagram": "^1.2",
-        "graylog2/gelf-php": "^1.5",
-        "symfony/process": "^4.1|^5.4",
-        "ext-json": "*"
+        "php": "^8.0",
+        "react/socket": "^1.12",
+        "react/datagram": "^1.9",
+        "graylog2/gelf-php": "^1.5|2.0",
+        "symfony/process": "^5.4|^6.0",
+        "ext-json": "*",
+        "ext-zlib": "*"
     },
     "require-dev": {
-        "keboola/coding-standard": ">=7.0.2",
-        "phpstan/phpstan": "^1.5",
-        "phpstan/phpstan-phpunit": "^1.0.0",
-        "phpunit/phpunit": "^9.3",
-        "ext-sockets": "*",
-        "ext-zlib": "*"
+        "keboola/coding-standard": ">=14.0",
+        "phpstan/phpstan": "^1.9",
+        "phpstan/phpstan-phpunit": "^1.3",
+        "phpstan/phpstan-symfony": "^1.2",
+        "phpunit/phpunit": "^9.5",
+        "sempro/phpunit-pretty-print": "^1.4",
+        "symfony/dotenv": "^5.4|^6.0",
+        "ext-sockets": "*"
```

Comment:
`ext-sockets` bude asi potreba i pro provoz, ne jen pro dev, ne?

Reply:
no není to uvedený v závislostech, ale v realitě to tak asi je, takže dávám do normálních requirements

Reply:
ne, tak ani v realu to tam fakt není potřeba